test/src/com/xilinx/rapidwright/device/TestNode.java

```diff
@@ -39,10 +40,10 @@ public void testGetDownhillUphillNodesUnique(String deviceName, String nodeName)
         Device device = Device.getDevice(deviceName);
         Node node = device.getNode(nodeName);
 
-        List<Node> downhill = node.getAllDownhillNodes();
```

Comment:
Can we continue to test for non-unique-ness using the existing API so that it'll stick out if anything changes?

Reply:
Sure, I've updated the test to check both APIs.

Reply:
Looks good (I can't approve my own PR)